packages/plugin-import/src/index.test.js

```diff
@@ -1,5 +1,9 @@
 jest.mock('fs-extra');
 
+jest.mock('@emdaer/core', () => async () => 'Hello, World!', {
```

Comment:
There shouldn't be a need for a mock implementation. You can just check if it was called like here:
https://github.com/emdaer/emdaer/pull/56/files#diff-9cfed617b4f3c8a2119339cb62ec388eR90

Reply:
You'll have to require it in after the `jest.mock` and so that you have something to pass to `expect`

Reply:
Forgot that `@emdaer/core` was in this one... and good call on not needing to mock the implementation